variant_remapping_tools/reads_to_remapped_variants.py

```diff
@@ -220,16 +227,23 @@ def process_bam_file(bam_file_path, output_file, out_failed_file, new_genome, fi
             if pass_basic_filtering(primary_group, secondary_group, primary_to_supplementary, counter, filter_align_with_secondary):
                 left_read, right_read = _order_reads(primary_group, primary_to_supplementary)
                 if pass_aligned_filtering(left_read, right_read, counter):
-                    counter['Remapped'] += 1
-                    varpos, new_ref, new_alts, ops = calculate_new_variant_definition(left_read, right_read, fasta)
-                    info = left_read.query_name.split('|')
-                    if info[7] != '.':
-                        info[7] += ';'.join(ops)
+                    varpos, new_ref, new_alts, ops, low_confidence = calculate_new_variant_definition(left_read, right_read, fasta)
+                    if not low_confidence:
+                        counter['Remapped'] += 1
+                        info = left_read.query_name.split('|')
+                        if info[7] != '.':
+                            info[7] += ';'.join(ops)
+                        else:
+                            info[7] = ';'.join(ops)
+                        outfile.write(
+                            '%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\n' % (left_read.reference_name, varpos, info[4], new_ref,
+                                                                  ','.join(new_alts), info[5], info[6], info[7]))
                     else:
-                        info[7] = ';'.join(ops)
-                    outfile.write(
-                        '%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\n' % (left_read.reference_name, varpos, info[4], new_ref,
-                                                              ','.join(new_alts), info[5], info[6], info[7]))
+                        # Currently the alignment is not precise enough to ensure that the allele change for INDEL and
+                        # novel reference allele are correct. So we skip them.
+                        # TODO: add realignment confirmation
```

Comment:
what's realignment confirmation?

Reply:
I'm not sure actually what it will be. 
I'm thinking some form of confirmation that the alleles (REF and ALTs) are the same before and after the remapping.
@tskir was suggesting Multiple Sequence Alignment.

Reply:
To make it more explanatory perhaps we could link the relevant issue in the TODO: https://github.com/EBIvariation/variant-remapping/issues/24

Reply:
I mean, the issue still just says “As described at the end of the Google document”, but I guess it's better than nothing :D

Reply:
Added reference to the issue and EVA ticket

Reply:
I've at least linked to the Google doc in question in the Github issue, for what it's worth :D